### PR TITLE
Fix `deprecated_new_in_comment_reference` for `material` library

### PR DESCRIPTION
**Pull request by guidezpl**
In a future version of the SDK, these will be flagged, fix them now.

### CODE REVIEW

File: packages/flutter/pubspec.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ description: A framework for writing Flutter applications
 homepage: https://flutter.dev
 
 environment:
-  sdk: ">=2.12.0-0 <3.0.0"
+  sdk: ">=2.17.0-0 <3.0.0"
```
Comment (darrenaustin) [inline packages/flutter/pubspec.yaml:6]: Was this bump in the min version intentional?

**Review by darrenaustin — COMMENTED** [review]
Looks like a nice cleanup. Just a quick question about the bump in version number in the pubspec.

File: packages/flutter/pubspec.yaml (hunk 1 above)
Comment (guidezpl) [reply packages/flutter/pubspec.yaml]: No, fixed!

**Review by darrenaustin — APPROVED** [review]
LGTM. Thx.

**Pull request merged**